### PR TITLE
[DPTOOLS-2735] Add method to load yaml file for connections

### PR DESCRIPTION
**Pull request by sherryzhao1006**
---
https://jira.lyft.net/browse/DPTOOLS-2735
Adding CLI to accept connection file and check and update the connection.

Test:
Run from devbox -- first time:
Screen Shot 2020-04-01 at 3 27 57 PM

Run from devbox -- second time:
Screen Shot 2020-04-01 at 4 09 26 PM

### CODE REVIEW

**Comment by sherryzhao1006** [comment]
👀 @youngyjd 
cc: @feng-tao

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -1418,6 +1418,46 @@ def connections(args):
...
+    if args.filepath:
+        with open(args.filepath, 'r') as stream:
+            connections_config = yaml.load(stream)
+        connections = {
+            connection['conn_id']: connection['conn_uri']
+            for connection in connections_config['connections']
+        }
+        # check conn_id and conn_uri were passed from yaml file.
+        missing_args = list()
+        for conn_id, conn_uri in connections.items():
```
Comment (youngyjd) [inline airflow/bin/cli.py:1430]: how about changing in host and port? https://airflow.lyft.net/connection/list/

Comment (sherryzhao1006) [reply airflow/bin/cli.py:1430]: Here:https://github.com/lyft/incubator-airflow/blob/airflowinfra/airflow/models/connection.py#L119. Connection itself will parse the uri we provide and set host and port. So if other parts changed, we still can update the uri and let Connection to parse the update host and port.

Comment (youngyjd) [reply airflow/bin/cli.py:1430]: I am ok with it. But remember to provide an example in connections.yaml and explain that host, port, password, etc needs to be set in url.

**Comment by youngyjd** [comment]
👍

File: airflow/bin/cli.py (hunk 1 above)
Comment (sherryzhao1006) [reply airflow/bin/cli.py:1430]: will do.

**Pull request merged**